crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -406,15 +406,15 @@ impl ConfigStoreBuilder {
     #[cfg(all(feature = "oxlint2", not(feature = "disable_oxlint2")))]
     fn load_external_plugin(
         oxlintrc_dir_path: &Path,
-        plugin_name: &str,
+        plugin_relative_path: &str,
         external_linter: &ExternalLinter,
         resolver: &Resolver,
     ) -> Result<(), ConfigBuilderError> {
         use crate::PluginLoadResult;
 
-        let resolved = resolver.resolve(oxlintrc_dir_path, plugin_name).map_err(|e| {
+        let resolved = resolver.resolve(oxlintrc_dir_path, plugin_relative_path).map_err(|e| {
```

Comment:
Is this really a path? I hoped that package.json entries will be resolved too.

Reply:
It can be either e.g. `my-plugin` or `../my-plugin.js`.

But you're right - "plugin name" isn't right, but "plugin relative path" also isn't the ideal description.

What is the right word? "specifier"?

Reply:
I've gone with "specifier". Shout if you can think of a better name.